ara/cli/host.py

```diff
@@ -62,19 +78,17 @@ def get_parser(self, prog_name):
         parser.add_argument(
             'host',
             metavar='<host>',
-            help='Host name or id to show',
+            help='Host id to show',
         )
         return parser
 
     def take_action(self, args):
-        try:
-            host = (models.Host.query
-                    .filter((models.Host.id == args.host) |
-                            (models.Host.name == args.host)).one())
-        except models.NoResultFound:
+        host = models.Host.query.get(args.host)
```

Comment:
I'm not sure I see the advantage there.  I mean, calling `.one()` does exactly that (raises an exception if more than one result is found).